packages/fx-core/src/plugins/resource/apim/services/apimService.ts

```diff
@@ -459,7 +459,7 @@ export class ApimService {
         operation.displayName,
         resourceType.displayName
       );
-      this.logger?.info(LogMessages.operationFailed(operation, resourceType, resourceId));
+      this.logger?.warning(LogMessages.operationFailed(operation, resourceType, resourceId));
```

Comment:
>logger?.warning [](http://example.com/codeflow?start=11&length=15)

Should it be `logger?.error` since this is an operation error?

Reply:
This log is inside the retry handler. If the operation failed after retry, another error log will be printed.

Reply:
I see. Thanks Xiaofu for the explaination.

Reply:
Got it, change the log level back to warning